tests/qos/files/qos.yml

```diff
@@ -1309,3 +1309,155 @@ qos_params:
                 cell_size: 208
             hdrm_pool_wm_multiplier: 4
             cell_size: 208
+    th3:
+        topo-any:
```

Comment:
hi vmittal:
     we are testing qos sai base on your change, but we meet some questions, can you give some advice?
     1: how you get TH3 qos.yml datas?
     2: where to get ptf-brcm?